src/API.jl

```diff
@@ -709,6 +709,22 @@ instantiate(; kwargs...) = instantiate(Context(); kwargs...)
 function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing,
                      update_registry::Bool=true, verbose::Bool=false, kwargs...)
     Context!(ctx; kwargs...)
+    if !isfile(ctx.env.project_file) && isfile(ctx.env.manifest_file)
+        if manifest == false
+            pkgerror("cannot instantiate without a project file with `manifest=false` option")
+        end
+        _manifest = Pkg.Types.read_manifest(manifestfile)
+        deps = Dict()
+        for (uuid, pkg) in _manifest
+            if pkg.name in keys(deps)
+                # TODO, query what package to put in Project when in interactive mode?
+                pkgerror("cannot instantiate a manifest without project file when the manifest has multiple packages with the same name ($(pkg.name))")
+            end
+            deps[pkg.name] = string(uuid)
+        end
+        Types.write_project(Dict("deps" => deps), "Project.toml")
```

Comment:
The project might not be in the currect directory:
```suggestion
        Types.write_project(Dict("deps" => deps), Types.find_project_file())
```

Reply:
Or I guess:
```suggestion
        Types.write_project(Dict("deps" => deps), ctx.env.project_file)
```

Reply:
Yeah, I accidentally left some of that assumption in, thanks.